util/linuxfw/nftables_runner.go

```diff
@@ -804,6 +804,32 @@ func (n *nftablesRunner) AddChains() error {
 	return n.conn.Flush()
 }
 
+// createDummyPostroutingChains is used to create dummy postrouting chains in
```

Comment:
"is used to"

rewrite in an active way to say what it does first, and then you can say how it's used in a second paragraph.